config/private.go

```diff
@@ -0,0 +1,6 @@
+package config
+
+// Private contains configurations which are private, adding it to be part of config without exposing to customers.
+type Private struct {
+	OAuthTokensPurgeInterval int
```

Comment:
May as well sort the one other `json:"-"` flag in here, and explicitly add `json:"-"` - we do not want the value to be encodeable into json, so we need to tag fields anyway